partiql-ast/src/main/java/org/partiql/ast/v1/expr/WindowFunction.java

```diff
@@ -15,8 +15,8 @@
 @EqualsAndHashCode(callSuper = false)
 public class WindowFunction extends AstEnum {
     public static final int UNKNOWN = 0;
-    public static final int LAG = 0;
-    public static final int LEAD = 0;
```

Comment:
(self-review) these enum variants were inadvertently all `0`. Adding the `name` method to `AstEnum` allowed us to catch this bug.